### PR TITLE
Snackbar: fixed position on mobile

### LINKED ISSUE #4994: [Bug]: Snackbar: fixed position on mobile
### Blazorise Version

1.3.1

### What Blazorise provider are you running on?

Bootstrap5

### Link to minimal reproduction, or a simple code snippet

https://blazorise.com/docs/extensions/snackbar

Both my app and the Blazorise Demo of the SnackbarStack have the issue so I will just point back to the demo.

### Steps to reproduce

Set browser device mode to mobile device like iPhone 12 pro.

Go to the "Stacked snackbars" section of the docs toward bottom of page.

Press the Primary button.

### What is expected?

A snackbar/toast should appear completely on screen at the Bottom End of the screen saying:

"Current time is: " + DateTime.Now

### What is actually happening?

It appears at Bottom but is cut off and placed at Bottom Start or there is just not enough room and it appears that way.

I've tried other positions and on mobile it doesn't seem to matter. The snackbar is cut in half with only a portion on screen aligned start/left.

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge, Safari, Firefox

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by steal4life**
closes #4994

Reason behind:

In the previous version the snackbar stack didn't have keyframes in its scss that now allow opening and closing animations... among other things they caused only half of the snackbar to be visible on mobile, so I made a few changes with the animations to avoid that behavior

### CODE REVIEW

**Comment by David-Moreira** [comment]
@steal4life 
Is this ready for review?

**Comment by steal4life** [comment]
Yes it is @David-Moreira

File: Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss [hunk 1]
```diff
@@ -304,7 +300,7 @@ $snackbar-padding-y-multi: 1.25rem !default;
 .snackbar-stack-top-start .snackbar-hide,
 .snackbar-stack-top-end .snackbar-hide {
     transition: all var(--stack-transition-duration, 1000);
-    animation: hideTop var(--stack-animation-duration, 500) forwards;
+    animation: hideTop var(--stack-animation-duration, 500) backwards;
```
Comment (David-Moreira) [inline Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss:303]: Why is the animation changed to backwards? WAs it wrong before? Or will this now produce a different behaviour that users might not be expecting on their apps?

**Review by David-Moreira — APPROVED** [review]

File: Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss (hunk 1 above)
Comment (stsrki) [reply Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss:303]: Same question from me. Even if I revert it to `forwards` the animation for stacked snackbars still doent's work. When snackbar hides it should hide from bottom to top when it is placed on top of the screen. At the moment it closes from top to bottom, which seems strange.

**Comment by steal4life** [comment]
@David-Moreira this solution works, but in case the snackbar is 50 percent of the screen height, then for some reason it will go down again, as long as it's smaller it should work

File: Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss [hunk 2]
```diff
@@ -297,93 +293,67 @@ $snackbar-padding-y-multi: 1.25rem !default;
...
     0% {
-        max-height: 0;
-        padding: 0;
-        transform: translate(-50%) translateY(-50px);
         opacity: 0;
     }
 
     100% {
-        max-height: 150px;
-        padding: $spacer-sm $spacer;
```
Comment (David-Moreira) [inline Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss:337]: Do we need to remove these?

**Review by David-Moreira — CHANGES_REQUESTED** [review]
It does not seem to sit at bottom end, should it not?

image

File: Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss (hunk 2 above)
Comment (steal4life) [reply Source/Extensions/Blazorise.Snackbar/Styles/blazorise.snackbar.scss:337]: I already removed that in last commit, do you see it?

**Review by stsrki — APPROVED** [review]

**Pull request merged**